src/frontend/view/view.ts

```diff
@@ -181,73 +184,42 @@ const viewErrorBanner = () => {
-    )
-    html.push(`
-      <div class="govuk-body">
-        <fieldset class="govuk-fieldset" ${
-          state.waiting && 'disabled="disabled"'
-        }>
-          <legend class="govuk-fieldset__legend">For each result, display:</legend>
-          <ul class="kg-checkboxes" id="show-fields">`)
-    html.push(
-      Object.keys(state.searchResults[0])
-        .map(
-          (key) => `
-            <li class="kg-checkboxes__item">
-              <input class="kg-checkboxes__input"
-                     data-interactive="true"
-                     type="checkbox" id="show-field-${key}"
-                ${state.showFields[key] ? 'checked' : ''}/>
-              <label for="show-field-${key}" class="kg-label kg-checkboxes__label">${fieldName(
-            key
-          )}</label>
-            </li>`
-        )
-        .join('')
-    )
-    html.push(`
-          </ul>
-        </fieldset>
-        <table id="results-table" class="govuk-table">
-          <tbody class="govuk-table__body">
-          <tr class="govuk-table__row">
-            <th scope="col" class="a11y-hidden">Page</th>`)
-    Object.keys(state.showFields).forEach((key) => {
-      if (state.showFields[key]) {
-        html.push(
-          `<th scope="col" class="govuk-table__header">${fieldName(key)}</th>`
-        )
-      }
-    })
-
-    recordsToShow.forEach((record, recordIndex) => {
-      html.push(`
-        <tr class="govuk-table__row">
-          <th class="a11y-hidden">${recordIndex}</th>`)
-      Object.keys(state.showFields).forEach((key) => {
-        if (state.showFields[key]) {
-          html.push(
-            `<td class="govuk-table__cell">${fieldFormat(
-              key,
-              record[key]
-            )}</td>`
-          )
-        }
-      })
-      html.push(`</tr>`)
-    })
-    html.push(`
-          </tbody>
-        </table>
-      </div>`)
-    return html.join('')
-  } else {
+  if (!state.searchResults || state.searchResults?.length <= 0) {
     return ''
   }
+  html.push(`
+    <div class="govuk-body">
+      <fieldset class="govuk-fieldset" ${
+        state.waiting && 'disabled="disabled"'
+      }>
+        <legend class="govuk-fieldset__legend">For each result, display:</legend>
+        <ul class="kg-checkboxes" id="show-fields">`)
+  html.push(
+    Object.keys(state.searchResults[0])
+      .map(
+        (key) => `
+          <li class="kg-checkboxes__item">
+            <input class="kg-checkboxes__input"
+                    data-interactive="true"
+                    type="checkbox" id="show-field-${key}"
+              ${state.showFields[key] ? 'checked' : ''}/>
+            <label for="show-field-${key}" class="kg-label kg-checkboxes__label">${fieldName(
+          key
+        )}</label>
+          </li>`
+      )
+      .join('')
+  )
+  html.push(`
+        </ul>
+      </fieldset>`)
+  html.push(
+    '<div id="results-grid-container" style="margin-top: 10px;" class="ag-theme-alpine"></div>'
```

Comment:
Can you take out this inline css and add to grid.scss instead or use the override class `govuk-!-margin-top-2` ?